release/models/wifi/openconfig-wifi-phy.yang

```diff
@@ -306,6 +321,35 @@ module openconfig-wifi-phy {
         "Do not perform scanning if any traffic received from an
         active Station in the past 100ms marked as AC_VO or AC_VI.";
     }
+
+    leaf wifi-protocol {
+      type identityref {
+        base oc-wifi-types:WIFI_PROTOCOL;
+      }
+      default "oc-wifi-types:WIFI_80211_AX";
```

Comment:
Is there an obvious choice as to why this should be the default?  Would it be better just to not specify any default and have it explicit?